plugin-default-impl/src/test/java/com/alibaba/nacos/plugin/auth/impl/persistence/ExternalUserPersistServiceImplTest.java

```diff
@@ -107,9 +107,12 @@ public void testFindUserByUsername() {
     
     @Test
     public void testGetUsers() {
-        Page<User> users = externalUserPersistService.getUsers(1, 10, "nacos");
+        int pageNo = 1;
+        int pageSize = 10;
+        Page<User> users = externalUserPersistService.getUsers(pageNo, pageSize, "nacos");
         
         Assert.assertNotNull(users);
+        Assert.assertTrue(users.getPageItems().size() <= pageSize);
     }
     
```

Comment:
The unit cases are not sufficient? since the original codes can also pass this UT.